poolmgr/gpm.go

```diff
@@ -86,9 +87,16 @@ func (gpm *GenericPoolManager) service() {
 			var err error
 			pool, ok := gpm.pools[tpr.CacheKey(&req.env.Metadata)]
 			if !ok {
+				var poolSize int32 = 3
+				switch req.env.Spec.AllowedFunctionsPerContainer {
+				case fission.AllowedFunctionsPerContainerInfinite:
+					poolSize = 1
+				}
+				log.Println("poolsize: ", poolSize)
+				log.Println("setting: ", req.env.Spec.AllowedFunctionsPerContainer)
```

Comment:
Are these logs intended for commit? If so, they should also have the name of the env, should be combined into one log message, and should have "functions per container" instead of "setting".